obspy/io/shapefile/core.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+from future.utils import native_str
+
+from obspy import Catalog
+from obspy.core.event import Origin, Magnitude
+from obspy.core.inventory import Inventory
+
+
+try:
```

Comment:
I generally prefer programs to not catch import errors and then reraise them with another description. Why not just let the original import error raise? At least to me its more clear whats going on then.